doc/release-notes/rl-2505.section.md

```diff
@@ -53,7 +53,9 @@
 
 ### Deprecations {#sec-nixpkgs-release-25.05-lib-deprecations}
 
-`functor` is an implementation detail and should not be relied upon, but since its status wasn't clear and it has had some use cases without alternatives, changes are being handled as gracefully as possible. Deprecations within functor:
-- `functor.wrapped` is now deprecated for some types and using it will give a warning with migration instructions. It is deprecated for these types:
+- `functor` is an implementation detail and should not be relied upon, but since its status wasn't clear and it has had some use cases without alternatives, changes are being handled as gracefully as possible. Deprecations within functor:
+  - `functor.wrapped` is now deprecated for some types and using it will give a warning with migration instructions. It is deprecated for these types:
     - `lib.types.attrsWith`
     - `lib.types.listOf`
+
+- Plasma 5 and Qt 5 based versions of associated software are deprecated in NixOS 25.05, and will be removed in NixOS 25.11. Users are encouraged to upgrade to Plasma 6.
```

Comment:
You mean qt5 versions of the plasma software not qt5 software in general, right?

Reply:
Yes, thus "associated".